apps/journeys-admin/src/components/ImageThumbnail/ImageThumbnail.stories.tsx

```diff
@@ -0,0 +1,93 @@
+import { Story, Meta } from '@storybook/react'
+import type { TreeBlock } from '@core/journeys/ui/block'
+import Box from '@mui/material/Box'
+
+import NoteAddIcon from '@mui/icons-material/NoteAdd'
+import {
+  GetJourney_journey_blocks_CardBlock as CardBlock,
+  GetJourney_journey_blocks_ImageBlock as ImageBlock
+} from '../../../__generated__/GetJourney'
+import { simpleComponentConfig } from '../../libs/storybook'
+import { ThemeMode } from '../../../__generated__/globalTypes'
+import { ImageThumbnail } from './ImageThumbnail'
+
+const ImageEditorStory = {
+  ...simpleComponentConfig,
+  component: ImageThumbnail,
+  title: 'Journeys-Admin/Editor/ImageThumbnail',
```

Comment:
Title should generally match the path the component is in
```suggestion
  title: 'Journeys-Admin/ImageThumbnail',
```